app/services/expects-scoring.js

```diff
@@ -39,38 +24,27 @@ export default Service.extend({
         return Object.values(groupBy(e => e.id)(expects))
     },
 
-    scorableCombinedExpectations(expects) {
-        return expects.filter(expect => this.isScorable(expect)).map(expect => this.replaceCombinedExpectDescription(expect))
-    },
-
     combineMultipleExpectations(expects) {
-        return this.groupById(expects).map(group => group.reduce((e1, e2) => this.combineExpectPair(e1, e2)))
+        return this.groupById(expects).map(this.expectationGroupingResult)
     },
 
-    combineExpectPair(expect1, expect2) {
-        return {
-            ...expect1,
-            ...expect2,
-            result: expect1.result || expect2.result
-        }
-    },
-
-    isScorable(expect) {
-        return !nonScorableExpectsIds.some(id => id === expect.id)
+    //The result of a grouping is an expectation that represents that the grouping passed or failed. 
+    //A grouping passes when at least an expectation passes, otherwise it fails.
+    expectationGroupingResult(expectationGrouping) {
+        const expectPasses = (e) => e.result
+        const possiblePassingExpect = find(expectPasses)(expectationGrouping)
+        const anyExpect = expectationGrouping[0]
+        return possiblePassingExpect || anyExpect
```

Comment:
Uh, claro, al pedo crear objetos nuevos!! :clap: :clap: :clap: :clap: 
Además me gustó el "represents", es una idea que me gusta más que la de "combine".

Ah, el  nombre "anyExpect" me hizo cosas raras en la cabeza, porque me esperaba un "any", y se me mezcló con la lógica del find. ¿Queda muy feo así?

```suggestion
    /**
     * @returns an expectation that represents the group. 
     * Its result is true if any expectation from the group passed.
    */
    expectationGroupingResult(expectationGrouping) {
        return expectationGrouping.find( e => e.result ) || expectationGrouping[0]
```

Yo sé que querés usar ramda, pero js ya tiene un find, y si no querés el find currificado ¿para qué ramda?

Reply:
No tengo idea de porque puse el find de ramda 👀